emily/lambda/build-lambda.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# TODO: replace this with a more organic build script that
+# automatically triggers when building the cdk.
+if [[ "$(uname -m)" == "arm64" ]]; then
+  cargo lambda build --release --output-format zip --arm64
+else
+  cargo lambda build --release --output-format zip
```

Comment:
Oh yeah I was just about to ask about why the default toolchain isn't used when building the a cargo lambda release, but then I saw the TODO 😄.

But we should include something in the README about installing `cargo-lambda` and the AWS CDK.

Reply:
> I was just about to ask about why the default toolchain isn't used when building the a cargo lambda release

I actually have no idea how to use rust toolchains. Before I look it up, do you have a two sentence explanation on what I should be doing?

Reply:
I don't think you should have to do anything at all to get _right_ build, so it's possible that I had the wrong model of what's going on here.

I think the disconnect for me was what the cargo-lambda folks were thinking here. Cargo will use the default toolchain when making a build unless you specify otherwise (like in the `rust-toolchan.toml` or in the command). Cargo-lambda is probably defaulting to build for whatever architecture is needed to run your Lambda on AWS. So maybe I'm just missing the memo.